css/component/button.css

```diff
@@ -0,0 +1,22 @@
+.btn{
+    border-radius: 10px;
+    cursor:pointer;
+}
+.btn-outline{
+    border:1px solid var(--primary-color);
+    background: transparent;
+    color:var(--primary-color)
+}
+.rm-border{
+    border:none;
+}
```

Comment:
Use better naming This rm looks bit awkward can be rem instead It seems to be much better understandability

Reply:
thanks for reviewing Akshat but I think using rem can create confusion as its unit for font size